glTF-Toolkit/inc/BufferBuilder.h

```diff
@@ -0,0 +1,119 @@
+/**************************************************************
+*                                                             *
+*  Copyright (c) Microsoft Corporation. All rights reserved.  *
```

Comment:
This shouldn't be here - we must wait for this to be implemented on Microsoft.glTF.CPP

Reply:
Understood, just started the Pull Request so we could get this review process going.